### ISSUE TITLE
unable to start on macOS, platform metadata area could not be written

### ISSUE THREAD

**Issue by scotthz** [cncgoko/Goko#56]
Goko will not start for me on macOS Mojave.

The stack backtrace contains the following message (see link below for complete log file).

> Caused by: java.lang.IllegalStateException: The platform metadata area could not be written: /private/var/folders/ks/kyzjq8fn3d12732z49tr3_5h005b65/T/AppTranslocation/51990801-4371-4C43-8419-8F9330B47A4A/d/Goko.app/Contents/MacOS/workspace/.metadata.  By default the platform writes its content
under the current working directory when the platform is launched.  Use the -data parameter to
specify a different content area for the platform.

I have not tried using the `-data` parameter because I don't know how to add command-line params when running an app on macOS.

Installed from goko-0.3.6-macosx.cocoa.x86_64.zip
Java version "1.8.0_171"
Log file: [`$HOME/.eclipse/1398918656_macosx_cocoa_x86_64/configuration/1543953314547.log`](https://github.com/cncgoko/Goko/files/2645968/1543953727432.log)

**Comment by cncgoko** [comment]
Hi @scotthz 

Not very familiar with Mac, but is there any way you can run it from a folder where you have write access ? 

But otherwise, the `-data` seems to do the trick. See this post on SO on how to access it: https://stackoverflow.com/questions/15454889/eclipse-ini-file-not-present-on-mac
The structure of the Goko package is the same as the described Eclipse package

**Comment by scotthz** [comment]
Thanks for the quick response. 

I am indeed trying to run Goko from a writeable folder. The pathname from the error message (starting with "/private/var/folders/...") comes from some opaque place deep in the JVM and macOS, and I'm not sure it can be configured. 

Regardless, I found another resolution to this issue. After a little more investigation I found a way to add CLI params when running an app on macOS. Instead of using the mouse to launch, one can use the `open` command in a shell window. The `--args` option allows app-specific parameters to be specified. So this command now allows me to launch Goko:

`open Goko.app --args -data $HOME/goko-data`

Hi @justinclift, it seems we travel in similar circles 😎
